### ISSUE TITLE
Blazor WASM: Small bug with error given on removed directory in relation to the _Imports.razor

### ISSUE THREAD

**Issue by Kayomez** [dotnet/aspnetcore#24965]
The error produced when there is an incorrect directory in the _Imports.razor file doesn't point you in the direction of the _Imports.razor file when clicked on. It seems to take you to seemingly random files. The description is fine mind you - I just have a tendency to not read things. Who needs reading when you got clicking!

Easy reproduce: move everything out of the Shared folder and into the root client folder > delete Shared folder.

**Comment by NTaylorMullen** [comment]
This is happening because technically we remap all usings to the current file (the ones included in the _Imports.razor); however, it sounds like the mapped line pragmas are not being respected.

@Kayomez  would you mind providing a repro application to investigate this further.

**Comment by Kayomez** [comment]
Makes sense. It seems to only bring up an error when you have an open tab for a given razor file. [Example repro.](https://github.com/Kayomez/_ImportsBug)
